sklearn/feature_extraction/_dict_vectorizer.py

```diff
@@ -224,6 +224,10 @@ def fit_transform(self, X, y=None):
         -------
         Xa : {array, sparse matrix}
             Feature vectors; always 2-d.
+
+        .. versionchanged:: 0.16
```

Comment:
This is not an API change, but a performance enhancement. It should not be documented here.